Docker_files/build.sh

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
+
+#################################################################################
+#                                                                               #
+# This script is for building generic performance container and push it to      #
```

Comment:
this is not part of the test, i add it to ocs-ci so we will have all in one place including the Dockerfile and a simple script to build the image(s).
yes this can be managed as a different github project, but it will be more complex, IMO.

Reply:
In that case it looks ok to me.